cmd/kor/root.go

```diff
@@ -52,8 +53,22 @@ func Execute() {
 	rootCmd.PersistentFlags().StringVar(&slackOpts.WebhookURL, "slack-webhook-url", "", "Slack webhook URL to send notifications to")
 	rootCmd.PersistentFlags().StringVar(&slackOpts.Channel, "slack-channel", "", "Slack channel to send notifications to. --slack-channel requires --slack-auth-token to be set.")
 	rootCmd.PersistentFlags().StringVar(&slackOpts.Token, "slack-auth-token", "", "Slack auth token to send notifications to. --slack-auth-token requires --slack-channel to be set.")
+	addFilterOptionsFlag(rootCmd, filterOptions)
+
+	if err := filterOptions.Validate(); err != nil {
```

Comment:
The validation will always fail since the parameters are not yet set.

Reply:
i fixed the issue in the `validate()` method. the problem was that optional flags (exclude-labels, --newer-than and --older-than) were checked any way even when not provided.